packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -699,9 +703,16 @@ const Dashboard: FC = () => {
 };
 
 const DashboardPage: FC = () => {
+    const { projectUuid } = useParams<{ projectUuid: string }>();
+    const { user } = useApp();
+    const dashboardCommentsCheck = useDashboardCommentsCheck(user?.data);
+
     useProfiler('Dashboard');
     return (
-        <DashboardProvider>
+        <DashboardProvider
+            projectUuid={projectUuid}
+            dashboardCommentsCheck={dashboardCommentsCheck}
```

Comment:
now, `DashboardProvider` knows if it can fetch dashboard comments. This is only passed to the `Dashboard`, so `Minimal` and `Embedded` remain OK